src/utils/RNPxlSearch.cpp

```diff
@@ -92,6 +92,25 @@ using namespace OpenMS;
 using namespace OpenMS::Internal;
 using namespace std;
 
+
+//-------------------------------------------------------------
+// Doxygen docu
+//-------------------------------------------------------------
+
+/**
+    @page UTILS_RNPxlSearch RNPxlSearch 
+
+    @brief Annotate RNA to peptide crosslinks in MS/MS spectra.
```

Comment:
@timosachsenberg can you expand the docu here at some point?

Reply:
will do